src/Function/Properties/Bijection.agda

```diff
@@ -30,16 +31,33 @@ private
     A B : Set a
     T S : Setoid a ℓ
 
+------------------------------------------------------------------------
+-- Conversion functions
+
+Bijection⇒Inverse : Bijection S T → Inverse S T
+Bijection⇒Inverse bij = record
+  { to        = to
+  ; from      = section
+  ; to-cong   = cong
+  ; from-cong = injective⇒section-cong surjection injective
+  ; inverse   = section-inverseˡ
+              , λ y≈to[x] → injective (Eq₂.trans (section-strictInverseˡ _) y≈to[x])
+  }
+  where open Bijection bij
+
+Bijection⇒Equivalence : Bijection T S → Equivalence T S
+Bijection⇒Equivalence = Inverse⇒Equivalence ∘ Bijection⇒Inverse
+
 ------------------------------------------------------------------------
 -- Setoid properties
 
 refl : Reflexive (Bijection {a} {ℓ})
 refl = Identity.bijection _
 
--- Can't prove full symmetry as we have no proof that the witness
--- produced by the surjection proof preserves equality
-sym-≡ : Bijection S (setoid B) → Bijection (setoid B) S
-sym-≡ = Symmetry.bijection-≡
+-- Now we *can* prove full symmetry as we now have a proof that
+-- the witness produced by the surjection proof preserves equality
+sym : Bijection S T → Bijection T S
+sym = Inverse⇒Bijection ∘ Symmetry.inverse ∘ Bijection⇒Inverse
```

Comment:
This change is great though!

Reply:
This refers to an out-of-date commit now?

Reply:
I had first managed to prove symmetry by going via `Inverse`, as above, but the current state of the PR now proves `sym` *directly*, thanks to `injective` `Surjection`s automatically satisfying `Congruent from`...